docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,67 @@
+# Getting Started on HPC systems
+
+These tutorials have been ran in in-person workshops on various HPC systems in the UK. It should be possible to run on any of the [supported systems](../systems.md), or to [set up the tools](../setup.md) on a local machine. If you have access to one of the systems we've previously used, this tutorial helps you get set up. Otherwise, please consult the [documentation](../install.md)
```

Comment:
" ran in in-person workshops", change to  "have been run at in-person workshops ...